### PR TITLE
Add CUDA 11.8 images

### PR DESCRIPTION
**Pull request by m3vaz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -136,6 +136,12 @@ jobs:
           DISTRO_NAME: "centos"
           DISTRO_VER: "7"
 
+          DOCKERIMAGE: linux-anvil-cuda
```
Comment (jaimergp) [inline .github/workflows/ci.yaml:139]:
```suggestion
        - DOCKERIMAGE: linux-anvil-cuda
```

**Comment by m3vaz** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
There's an error I am working on at #220. Stay tuned :)

**Comment by leofang** [comment]
@conda-forge-admin, please restart ci

**Comment by leofang** [comment]
oh this repo probably does not have the bot set up..

**Comment by hmaarrfk** [comment]
i thought we were to support to wait to move on this cc @beckermr

**Comment by beckermr** [comment]
Yes do not merge.

**Review by beckermr — CHANGES_REQUESTED** [review]
Do not merge.

**Comment by m3vaz** [comment]
From internal discussion, closing until further notice.

**Pull request closed without merging**